pages/common/siege.md

```diff
@@ -0,0 +1,28 @@
+# siege
+
+> HTTP loadtesting and benchmarking tool.
+> More Information: <https://www.joedog.org/siege-manual/>.
+
+- Test a url with default settings:
+
+`siege https://www.google.com`
```

Comment:
Missing token syntax? Also, maybe this would be more consistent with other pages?
```suggestion
`siege {{https://example.com}}`
```

Reply:
My bad, missing the token syntax in the style guide. Updated in the latest commit